gdal/swig/python/osgeo/gdal_array.py

```diff
@@ -263,12 +263,10 @@ def DatasetReadAsArray(ds, xoff=0, yoff=0, win_xsize=None, win_ysize=None, buf_o
         interleave = True
         xdim = 2
         ydim = 1
-        bdim = 0
```

Comment:
each time you touch swig/python/osgeo/* files you need to do changes in the sources files in swig/include/